packages/components-css/skip-link-css/src/_mixin.scss

```diff
@@ -1,3 +1,48 @@
-/* stylelint-disable-next-line block-no-empty */
 @mixin nl-skip-link {
+  /* Calculate the `max-inline-size` to always leave room for the focus ring */
+  align-items: center;
+  background-color: var(--nl-skip-link-background-color, CanvasText);
+  box-sizing: border-box;
+  color: var(--nl-skip-link-color, LinkText);
+  display: inline-flex;
+  font-size: var(--nl-skip-link-font-size);
+  justify-content: center;
+  line-height: var(--nl-skip-link-line-height);
+  max-inline-size: calc(100% - 2 * var(--_nl-skip-link-inset, 0));
+  min-block-size: var(--nl-skip-link-min-block-size, 44px);
+  min-inline-size: var(--nl-skip-link-min-inline-size, 44px);
+  padding-block: var(--nl-skip-link-padding-block);
+  padding-inline: var(--nl-skip-link-padding-inline);
+  text-decoration-thickness: max(var(--nl-skip-link-text-decoration-thickness, 0px), initial);
+  text-underline-offset: var(--nl-skip-link-text-underline-offset, initial);
+}
+
+@mixin nl-skip-link--focus-visible {
+  background-color: var(--nl-skip-link-focus-visible-background-color);
```

Comment:
Ik zou willen voorstellen om deze stijl weg te laten totdat we een ei hebben gelegd over de algemene focus-visible aanpak.

Reply:
done